.env

```diff
@@ -0,0 +1 @@
+REACT_APP_TMDB_API_KEY = '9efb0e1e640a5bfe13aa9ef51a060265'
```

Comment:
🔴 boy boy boy .... please add it to you .gitignore file 🚨

Reply:
completely forgot, thanks 😄